README.md

```diff
@@ -19,16 +19,6 @@ The available features will depend on the features exposed by each dispatched mo
 
 ##### MANO enabler
 - Validation service: Validate VNFD or NSD as a standalone service
-- Onboard VNFD
-- List VNFDs
-- Retrieve single VNFD
-- Delete VNFD
-- Update VNFD
-- Onboard NSD
-- List NSDs
-- Retrieve single NSD
-- Delete NSD
-- Update NSD
```

Comment:
These features don't disappear, If they don't belong to the MANO, they must be somewhere else